README.md

```diff
@@ -62,6 +81,102 @@ Or, in order to start using the `offline` mode:
 --port=9092 --offline
 ```
 
+## Setup a database
+
+In order to support historical balances lookup, Rosetta has to connect to an Observer whose database contains _non-pruned accounts tries_. Such databases can be re-built locally or downloaded from the Internet.
```

Comment:
downloaded from Elrond public archive, whose link can be obtained from Elrond Developers tg ?

Reply:
Adjusted.